Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -179,6 +179,6 @@ int ApplicationMainStart(const char* principalName,
     return 0;
 }
 
-void SetTemporaryFolder(const char* folder) {
-    NSSetTemporaryDirectory([NSString stringWithCString:folder]);
+void SetTemporaryFolder(const wchar_t* folder) {
+    NSSetTemporaryDirectory([NSString stringWithCharacters:(const unichar*)folder length:wcslen(folder)]);
```

Comment:
Darn. If there was some way to get a HSTRING out of the layer above this one, we could pass it via the NSString HSTRING bridge. That'd save us a whole copy :)